ngi_pipeline/engines/sarek/local_process_tracking.py

```diff
@@ -237,3 +239,10 @@ def remove_analysis(self, force=False):
         else:
             msg = "{} skipped".format(msg)
         self.log.debug(msg)
+
+    def cleanup(self):
+        # only cleanup if the process exited successfully
+        if self.process_status != ProcessExitStatusSuccessful:
+            return
```

Comment:
I am pretty sure if the process is not successful it is logged somewhere, but do you think it would be useful to have log here as well ?

Reply:
good point, I agree. I've been a bit lazy with the error handling and logging, I should probably review that at some point. I'll push a fix for this now though.